### PR TITLE
K8s hellonode scenario

### PR DESCRIPTION
**Pull request by hunchback**
simply make sure that k8s tests run, when developing the tests I discovered some issues with links:
- cluster --> node
- node --> pod

So for now (until fixing the k8s probe) these were eliminated from the test scenario

### CODE REVIEW

File: tests/k8s_test.go [hunk 1]
```diff
@@ -61,54 +62,77 @@ var (
...
+	args := []interface{}{"Namespace", m["Namespace"], "Name", m["Name"]}
+	args = append(args, args1...)
+	return makeHasArgsType(m["Type"], args...)
+}
+
+func queryNodeCreation(c *CheckContext, query g.QueryString) (node *graph.Node, err error) {
+	err = common.Retry(func() error {
+		const expectedNumNodes = 1
+
+		fmt.Printf("Gremlin Query: %s\n", query)
```
Comment (safchain) [inline tests/k8s_test.go]: t.Logf instead of Printf passing the `t` variable to the function

File: tests/k8s_test.go [hunk 2]
```diff
@@ -61,54 +62,77 @@ var (
...
+
+		if len(nodes) != expectedNumNodes {
+			return fmt.Errorf("Ran '%s', expected %d node, got %d nodes: %+v", query, expectedNumNodes, len(nodes), nodes)
+		}
+
+		if expectedNumNodes > 0 {
+			node = nodes[0]
+		}
+		return nil
+	}, k8sRetry, k8sDelaySeconds*time.Second)
```
Comment (safchain) [inline tests/k8s_test.go]: as k8sDelaySeconds says that it is in second I would put the `8*time.Second` in the constant definition line 47

File: tests/k8s_test.go (hunk 1 above)
Comment (hunchback) [reply tests/k8s_test.go]: fixed

File: tests/k8s_test.go (hunk 2 above)
Comment (hunchback) [reply tests/k8s_test.go]: fixed and renamed `k8sDelaySeconds` --> `k8sDelay`

**Comment by safchain** [comment]
rerun scale-tests

**Pull request merged**